app/javascript/app/components/country-ndc-overview/country-ndc-overview-styles.scss

```diff
@@ -21,13 +21,9 @@
 .actions {
   @extend %grid;
 
-  @include msGridColumns(1fr, 20fr, 32fr);
+  @include msGridColumns(45px, 1fr, 47%);
```

Comment:
Amazingly this seems to work! I think the 47% is going to pollute the grid at lower screen sizes as it calculates it from true 100% width, so the gutter gets ignored sometimes.